apps/tailwind-components/components/form/AddModal.vue

```diff
@@ -55,13 +55,14 @@
         <FormFields
           v-if="visible"
           ref="formFields"
-          :schemaId="schemaId"
-          :metadata="metadata"
-          :sections="sections"
+          :row-key="rowKey"
+          :columns="visibleColumns"
           :constantValues="constantValues"
-          v-model:errors="errorMap"
+          :errorMap="errorMap"
           v-model="formValues"
-          @update:active-chapter-id="activeChapterId = $event"
+          @update="onUpdateColumn"
```

Comment:
`@update` would be useful for removing errors immediately but it would also trigger validation before the input is complete

Reply:
turns out te be another issue with the Date, fixed